src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -213,7 +213,7 @@
     <value>Cancel</value>
   </data>
   <data name="FileChanged.Message" xml:space="preserve">
-    <value>Hosts file has changed by another application. Do you want to reload it?</value>
+    <value>Hosts file was modified externally.</value>
```

Comment:
```suggestion
    <value>Hosts file was modified by another application.</value>
```

I think it's more clear. The question is redundant.

Reply:
Hm, semantics. 🤔 If I open a file, edit the content and then save it; did I change it or did the app? I could've used another text editor myself and saying an app did it would make me feel passed.